skimage/future/manual_segmentation.py

```diff
@@ -134,13 +139,19 @@ def _extend_polygon(event):
     labels = (_mask_from_vertices(vertices, image.shape[:2], i)
               for i, vertices in enumerate(list_of_vertex_lists, start=1))
     if return_all:
-        return np.stack(labels)
+        return np.stack(labels), list_of_vertex_lists
     else:
-        return reduce(np.maximum, labels, np.broadcast_to(0, image.shape[:2]))
+        return reduce(
+            np.maximum,
+            labels,
+            np.broadcast_to(0, image.shape[:2])
+        ),
+        list_of_vertex_lists
 
 
 def manual_lasso_segmentation(image, alpha=0.4, return_all=False):
-    """Return a label image based on freeform selections made with the mouse.
+    """Return a label image  and list of contours based on freeform selections
```

Comment:
Keep it a one-liner. If more detailed description is required, put it 2 lines below.
Also, fix a double whitespace.